client/components/product-expiration/index.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+interface Moment {
+	isValid: () => boolean;
+	format: ( arg0: string ) => string;
```

Comment:
Should `arg0` be called `format` or `dateFormat` instead?

Reply:
Fixed in 16feb54d14598303f399cc90336be44c634d157b